ui/components/ConfirmationModal.js

```diff
@@ -155,13 +155,20 @@ function ConfirmationMsg(props) {
   const { classes, open, handleClose, submit,
     selectedK8sContexts, k8scontext, title, validationBody, setK8sContexts, enqueueSnackbar, closeSnackbar, componentCount, tab, isVerify } = props
 
-  const [contexts, setContexts] = useState(k8scontext);
   const [tabVal, setTabVal] = useState(tab);
+  const [disabled, setDisabled] = useState(true);
+
+  let isDisabled = typeof selectedK8sContexts.length === "undefined" || selectedK8sContexts.length === 0
```

Comment:
How is this not a global variable? is this something that v0.7.0's approach to state management will address, @sudo-NithishKarthik?

Reply:
`const [disabled, setDisabled] = useState(true)` In a sense it is a global variable, as wherever we perform an operation that requires context, this modal is present. And no other component requires it

Reply:
@MUzairS15 are you aware of how this will change when there is global state management in v0.7.0? What note(s) do we need to take now in preparation?